espnet2/tts/gradtts/utils.py

```diff
@@ -0,0 +1,46 @@
+import torch
+
+
+def sequence_mask(length, max_length=None):
```

Comment:
might be better to use espnet torch_utils functions for the masking to keep the consistency